msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,333 @@
-* All apps must have a visual tour, such as **Take a Tour** or an **App Guide** in its configuration screen that talks about the app features and necessary integration in the following places:
-    * The store listing page (Long Description).
-    * Tab configuration screen.
-    * Welcome message for a bot.
-    * App source metadata.
-    * Connector configuration screen.
+* The name must include terms relevant to your users.
+* It must not copy the name of an existing app in the store or other offer in the commercial marketplace.
+* Common nouns must be prefixed or suffixed with the publisher’s name (for example, “XYZ Tasks” rather than “Tasks”)
+* It must not use **Teams** or other Microsoft product names that could falsely indicate co-branding or co-selling. (For more information about referencing Microsoft software, products, and services, see the [Microsoft Trademark and Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general)).
+* If your app is part of an official partnership with Microsoft, the name of your app must come first (for example, Salesforce Connector for Microsoft Teams).
 
-* The visual tour can be a  video, screenshot, a link to a static tab with app details. All these references must be within the Teams environment.
 
-    ![Sample App 1](../../../../assets/images/faq/Sampleapp1.png)
-    ![Sample App 2](../../../../assets/images/faq/Sampleapp2.png)
+### 1.2 Suitable for workplace consumption
 
-* Increment your app version number in the manifest if you make any manifest changes to your submission.
-* The app must not take users out of Teams for core user scenarios. Link targets in apps must not link to an external browser. Link targets must link to div elements contained within Teams, for example, task modules and tabs. 
-* Using task modules or tabs is suggested to display information to users within Teams.
-* All core and non-core scenarios must be completed within the Teams environment except for:
-  * Privacy Policy
-  * Terms Of Use (TOU)
-  * Website link
-  * Sign-up process
+Your app can facilitate group collaboration, improve an individual's productivity, or both. This includes apps intended for team bonding and socializing. Such apps should not require intense time investment or perceptively impact productivity. These apps must be collaborative and designed for multiple participants. 
 
-* Personal apps enable users to share content from a personal app experience with other team members.
+All content should be suitable for general workplace consumption and must abide by all restrictions listed in the [commercial marketplace certification policies](https://docs.microsoft.com/legal/marketplace/certification-policies#10010-inappropriate-content). Content related to religion, politics, gambling, and prolonged entertainment is prohibited.
```

Comment:
Rearranging the content - 
All content should be suitable for general workplace consumption and must abide by all restrictions listed in the commercial marketplace certification policies. Content related to religion, politics, gambling, and prolonged entertainment is prohibited.

Your app must facilitate group collaboration or improve an individual's productivity, or both. Apps intended for team bonding and socializing should not require intense time investment or perceptively impact productivity. These apps must be collaborative and designed for multiple participants.